pkg/apis/internal.acorn.io/v1/appinstance.go

```diff
@@ -68,7 +68,7 @@ type AppInstanceSpec struct {
 	Links           []ServiceBinding `json:"services,omitempty"`
 	Ports           []PortBinding    `json:"ports,omitempty"`
 	DeployArgs      GenericMap       `json:"deployArgs,omitempty"`
-	Permissions     *Permissions     `json:"permissions,omitempty"`
+	Permissions     []*Permissions   `json:"permissions,omitempty"`
```

Comment:
I'll probably need convinced that you need []*Permissions instead of []Permissions

Reply:
Yeah, that's a good point. Main reason I made it be `[]*Permissions` is because prior it was `*Permissions`. I believe that it was this way before because it allows us to do `nil` checks when interacting with `Permissions`. That being said, I'm not entirely sure that's worth it.

Reply:
Seeing how its used (thus far), I think the right thing to do is to drop the `*`. Where nil checks were needed before, you're now iterating through a loop, so i dont think its necessary